packages/mobile/src/screens/favorites-screen/AlbumsTab.tsx

```diff
@@ -39,6 +49,18 @@ export const AlbumsTab = () => {
     }
   }, [isReachable, hasMore, fetchMore])
 
+  const selectedCategory = useSelector(getSelectedCategory)
+  let emptyTabText: string
+  if (selectedCategory === LibraryCategory.All) {
+    emptyTabText = messages.emptyAlbumAllText
+  } else if (selectedCategory === LibraryCategory.Favorite) {
+    emptyTabText = messages.emptyAlbumFavoritesText
+  } else if (selectedCategory === LibraryCategory.Repost) {
+    emptyTabText = messages.emptyAlbumRepostsText
+  } else {
+    emptyTabText = messages.emptyAlbumPurchasedText
+  }
```

Comment:
not a big deal at all, but I wish we had a better pattern for this (not just here, but we often have a message type that varies based on an enum)

maybe a `getEmptyMessageByCategory` function or map? Could even make the messages object keyed by the enum? Maybe over-engineering 😅